modules/ref-supported-runtimes.adoc

```diff
@@ -0,0 +1,80 @@
+:_module-type: REFERENCE
+
+[id='supported-runtimes_{context}']
```

Comment:
Ensure that the ID matches the module title:

[id='supported-model-serving-runtimes_{context}']

Reply:
```suggestion
[id='supported-model-serving-runtimes_{context}']
```